.github/CODEOWNERS

```diff
@@ -64,6 +64,7 @@
 /src/compute-types                  @MaterializeInc/cluster
 /src/controller                     @MaterializeInc/cluster
 /src/controller-types               @MaterializeInc/cluster
+/src/durable-cache                  @aljoscha @danhhz @jkosh44
```

Comment:
I think we should find a team to own this. Feels like maybe adapter? Feels like it's about time to add @aljoscha to the adapter team, if he's up for it.

Reply:
I gave it to the adapter.